committee-steering/governance/README.md

```diff
@@ -43,7 +43,7 @@ As part of this we will define roles for the [OARP] process (Owners, Approvers,
   If more substantial changes are desired it is advisable to socialize those before drafting a PR.
     - The steering committee will be looking to ensure the scope of the SIG as represented in the charter is reasonable (and within the scope of Kubernetes) and that processes are fair.
 - For large changes alert the rest of the Kubernetes community (Participants) as the scope of the changes becomes clear.
-  Sending mail to [dev@kubernetes.io] and/or announcing at the community meeting are a good ways to do this.
+  Sending mail to [dev@kubernetes.io] and/or announcing it in the #kubernetes-contributors slack channel.
```

Comment:
I don't think slack is a sufficient way to announce large scale changes. Perhaps we just leave it to the mailing list?

Reply:
I'm good with that, I meant it more as an optional thing to mention the change in there. the mailing list should ALWAYS get the notifications.

Reply:
just changed and/or to and (optionally)